recipes/cryptominisat/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "cryptominisat" %}
+{% set version = "5.0.2" %}
+{% set sha256 = "d1815338d0563cc4132b4a72540f1c3485cd3c9efa3b38af4b2ebf99e0c567c7" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/msoos/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - 0001-Use-CFLAGS-LDFLAGS-as-reported-by-sysconfig.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - python
+    - setuptools
+    - boost
```

Comment:
Change this to `boost-cpp 1.66.0`

Reply:
Add `zlib 1.2.11`

Reply:
Why do I have to pin these?

Reply:
1.66.0 and 1.55.1 have different ABI and sonames. similar for zlib